clustermesh-apiserver/users_mgmt.go

```diff
@@ -57,6 +58,7 @@ type usersConfigFile struct {
 
 type usersManager struct {
 	UsersManagementConfig
+	cmtypes.ClusterIDName
```

Comment:
Does this have to be anonymous? If not we could dry up the naming...

Reply:
I've removed all anonymous usages of this struct to dry up the naming.